segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/filesystem/FileSystemStorage.java

```diff
@@ -72,14 +67,11 @@
  * current owner. Once the earlier owner received this notification, it stops writing to the segment.
  */
 @Slf4j
-public class FileSystemStorage implements Storage {
-    private static final int NUM_RETRIES = 3;
+public class FileSystemStorage extends IdempotentStorageBase {
```

Comment:
This change is result of CR comment to move common functionality between FileSystemStorage implementation and ExtendedS3Storage implementation in one place.